src/layerInfos/TarifverbundkarteTopicInfo/TarifVerbundLegend.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { makeStyles } from '@material-ui/core';
+
+import TarifverbundPartner from '../../components/TarifverbundPartner';
+
+const useStyles = makeStyles(() => {
+  return {
+    legend: {
+      display: 'grid',
+      gridTemplateColumns: (props) =>
+        props.isMobile ? '1fr 1fr' : '200px 200px',
+      gap: 5,
+    },
+  };
+});
+
+const useFetchVerbunde = (vectorTilesUrl) => {
+  const [verbunde, setVerbunde] = useState([]);
+  useEffect(() => {
+    fetch(`${vectorTilesUrl}/styles/ch.sbb.tarifverbund/style.json`)
```

Comment:
Are you sure we need this fetch you already have the style using the mapbox map directkly(accessing via the layer)

Reply:
No that won't work I think, because the User can view the Tarifverbundkarte LayerInfo even when the Topic is not active. If the Topic is not active, the Layer mbMap tag is undefined